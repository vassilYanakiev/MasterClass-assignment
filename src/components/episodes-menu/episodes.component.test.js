import {shallow} from 'enzyme';
import React from 'react';
import EpisodesItem from './episodes.component.jsx' ;


it('expected to render component',()=>{

    expect(shallow(<EpisodesItem/>)).toMatchSnapshot();
})