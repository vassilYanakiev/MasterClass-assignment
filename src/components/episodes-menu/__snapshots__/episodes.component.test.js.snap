// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expected to render component 1`] = `ShallowWrapper {}`;
